src/Factory_TEST.cc

```diff
@@ -60,6 +60,15 @@ TEST(FactoryTest, New)
 
   msg = msgs::Factory::New<msgs::Vector3d>(".gz.msgs.Vector3d");
   EXPECT_TRUE(msg.get() != nullptr);
+
+  unsetenv("GZ_DESCRIPTOR_PATH");
+  unsetenv("IGN_DESCRIPTOR_PATH");
+
+  msg = msgs::Factory::New<msgs::Vector3d>("gz.msgs.Vector3d");
+  EXPECT_TRUE(msg.get() != nullptr);
```

Comment:
This is slightly better to read when it fails, because it prints both values.

```suggestion
  EXPECT_NE(msg, nullptr);
```

And after I wrote this I realized you're just following the existing pattern. No biggie.

I was wondering what this test is checking though.

Reply:
I'm trying to cover the line cos code coverage is complaining :(

Reply:
How do I retrigger the bot?

Reply:
Ahh I see. The line still shows uncovered to me. I believe this is only initialized at static initialization, so if you change the env vars later it won't affect it. I think it's ok to leave those lines uncovered by tests.

Reply:
So, I think we should just remove those new test lines since they're not doing anything new.